### PR TITLE
SRT writer

### LINKED ISSUE #112: Implement SRT writer

### PR DESCRIPTION
**Pull request by valnoel**
Closes #112

### CODE REVIEW

File: src/main/python/ttconv/model.py [hunk 1]
```diff
@@ -300,16 +300,16 @@ def set_id(self, element_id: str):
   
   # style properties
 
-  def has_style(self, style_prop: StyleProperty) -> bool:
+  def has_style(self, style_prop: typing.Type[StyleProperty]) -> bool:
```
Comment (palemieux) [inline src/main/python/ttconv/model.py:303]: Why is this needed? Just curious.

**Comment by palemieux** [comment]
@valnoel Can you review https://github.com/sandflow/ttconv/issues/137?

File: src/main/python/ttconv/model.py (hunk 1 above)
Comment (valnoel) [reply src/main/python/ttconv/model.py:303]: Because the `style_prop` passed here is a StyleProperty type, not an instance of it:
We call
```python
element.has_style(Color)
```
instead of
```python
element.has_style(Color())
```

**Review by palemieux — COMMENTED** [review]
Can you check why converting the following to SRT is super slow?

https://raw.githubusercontent.com/Comcast/caption-inspector/master/test/media/Plan9fromOuterSpace.scc

**Review by palemieux — APPROVED** [review]
